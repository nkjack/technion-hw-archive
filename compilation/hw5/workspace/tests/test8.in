int Func(int x)
@pre(x > 0)
{
	printi(x);
	return x;
}
void maxTestsParamReturn() {
	int x = Func(5);
	print("\n");
	printi(x);
}

void main() {
     maxTestsParamReturn();
     print("\nI hate this course\n");
}
