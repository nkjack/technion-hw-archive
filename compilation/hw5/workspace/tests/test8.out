Loaded: ./exceptions.s
5
5
I hate this course
